cmd/rqlite/main.go

```diff
@@ -93,6 +95,12 @@ func main() {
 		}
 		term.Close()
 
+		hosts := createHostList(argv)
+		recoveringClient := httpcl.NewClient(client, hosts,
```

Comment:
I know naming is hard :-) but I would just call it `client`.

Reply:
It's just a smarter client, not a "recovering client".

Reply:
client is already declared before in the same scope.

and you are right about the `recovering` part, let me find some better name for it :)